catroid/src/main/java/org/catrobat/catroid/utils/Utils.java

```diff
@@ -639,4 +640,58 @@ public void onReceive(Context context, Intent intent) {
 			}
 		}, null, Activity.RESULT_OK, null, null);
 	}
+
+	public static List<PickableDrum> getPickableDrums() {
+		List<PickableDrum> drums = new ArrayList<PickableDrum>();
+
+		drums.add(new PickableDrum("Acoustic Bass Drum", 35));
```

Comment:
All these English names of the instruments (down to line 693) should be in strings.xml so that they can be translated via CrowdIn on https://translate.catrobat.org/ by our translators.